lisa/operating_system.py

```diff
@@ -1814,6 +1814,35 @@ def _update_packages(self, packages: Optional[List[str]] = None) -> None:
         if self._no_repo_enabled.search(output):
             raise RepoNotExistException(self._node.os)
 
+    def grow_os_partition_size(self) -> None:
+        from lisa.tools import Mount
+
+        pv_result = self._node.execute("pvscan -s", sudo=True, shell=True).stdout
```

Comment:
What's the pvscan? Does it exist on all distros?

Reply:
I don't think it exists outside RHEL 
https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/6/html/logical_volume_manager_administration/physvol_display

I tested in multiple version of RHEL (8.0, 8.3, 9.0, 9.3). the tool was available in all of them

The implementation is inside OS class (Redhat), so that should be alright ?